xmtp/src/client.rs

```diff
@@ -20,7 +24,8 @@ where
     pub api_client: A,
     pub network: Network,
     pub persistence: NamespacedPersistence<P>,
-    pub account: VmacAccount,
+    // TODO: Make account private. Just need to figure out how to access for tests
+    pub account: Account,
```

Comment:
You can use fine grain visibility scoping https://doc.rust-lang.org/reference/visibility-and-privacy.html

You could use `pub(self)` if the test are contained in the same module.

Reply:
Did not know `pub(self)` was a thing in Rust. That sounds very useful.